docs/nodes/bridge-node.md

```diff
@@ -5,6 +5,9 @@ This tutorial will go over the steps to setting up your Celestia Bridge node.
 Bridge nodes connect the data availability layer and the consensus layer
 while also having the option of becoming a validator.
 
+Validators do not have to run bridge nodes, but they should because it relays
+blocks to the DA network.
```

Comment:
I'm confused because 
https://github.com/celestiaorg/docs/blob/542d0ca2b696d7dcf8cd62e3b7c9811ddac31564/docs/nodes/overview.md?plain=1#L9-L10 

suggests that validators do need to run bridge nodes 🤷‍♂️

Reply:
If validators don't need to run bridge nodes, then perhaps a better line in the overview page would be:

```
* [Validator node](./validator-node.md): This type of node participates in consensus by producing and voting on blocks.
```

If so, then happy to merge this PR as-is and create an issue / PR to fix the overview.md page.

Reply:
Hmm, let me cc @renaynay to see if she can maybe provide more insight here.

Reply:
cc @Wondertan any insight on this discussion / description around giving more information about bridge nodes?

Reply:
Validators and bridge nodes are separate node types.

Bridge nodes need a connection to a celestia-core node (that means either validator or non-validating tendermint full node)

We should remove the line mentioned here https://github.com/celestiaorg/docs/pull/167/files#r931620621 from the docs because it is confusing. 

Validators *do not* have to run bridge nodes, but are encouraged to in order to relay blocks to the DA network.

Reply:
The reason we recommend validators to do it is because running a bridge node requires an RPC connection to a validating || non-validating tendermint node, and since validators are already running validators, they can feed their bridge node the RPC address of their own validator and run them in tandem.